tests/test_backups.py

```diff
@@ -78,29 +78,41 @@ def fixture_t4c_backup_container(
 )
 def test_model_backup_happy(
     t4c_backup_container: containers.Container,
+    git_ip_addr: str,
     git_http_port: str,
     tmp_path: pathlib.Path,
 ):
     conftest.wait_for_container(
-        t4c_backup_container,
-        "Import of model from TeamForCapella server finished",
+        t4c_backup_container, "Backup of model finished"
     )
 
     git_path = tmp_path / "test-git-data"
     git_path.mkdir()
 
+    if conftest.DOCKER_NETWORK == "host":
+        git_ip_addr = "127.0.0.1"
```

Comment:
I believe we cannot simply move it, as we need to consider two scenarios. When using the Git server within another container (e.g., our backup image, which clones, commits, and pushes changes to the Git server), we require the actual Docker IP address of the Git server. Conversely, we cannot use the Docker IP address when operating on the host network. If I am overlooking something, please let me know, as it would simplify the process in several aspects.